src/reduxStore.jsx

```diff
@@ -1,8 +1,20 @@
 import { createStore, applyMiddleware, combineReducers } from 'redux';
 import thunk from 'redux-thunk';
-import transactions from './Stats/reducer';
+import { persistStore, persistReducer } from 'redux-persist';
+import storage from 'redux-persist/lib/storage';
+import autoMergeLevel2 from 'redux-persist/lib/stateReconciler/autoMergeLevel2';
+import transactionReducer from './Stats/reducer';
 
-const mainReducer = combineReducers({
-  transactions,
+const persistConfig = {
+  key: 'root',
+  storage,
+  stateReconciler: autoMergeLevel2,
+};
+
+const rootReducer = combineReducers({
+  transaction: transactionReducer,
```

Comment:
it was transactions, do not change it, if it is not related to this story

Reply:
fixed